### ISSUE TITLE
Process "csgo.exe" not found!

### ISSUE THREAD

**Issue by Lenosis** [KyeOnDiscord/csgo-glow-demo-py#1]
Hey, 

Been tryna play around with it for a little, haven't had much luck. I am pretty new to programming so would appreciate some sort of help!!

![image](https://user-images.githubusercontent.com/123729374/215052902-abbc4266-ee8a-4342-8c2f-966313e84aec.png)

Thanks for any help!

**Comment by KyeOnDiscord** [comment]
As you can see it written in plain English, it says csgo is not opened.

**Comment by Lenosis** [comment]
> As you can see it written in plain English, it says csgo is not opened.

Ahh damn haha! Sorry!!


Do you reckon this would work without safe mode on?

By safe mode I mean "-insecure"

**Comment by KyeOnDiscord** [comment]
It'll work but you'll probably get banned. Use it at your own risk 🤓

**Comment by BillyHri** [comment]
Try reinstalling windows, this is a rare issue that can happen.
